src/Core/src/Platform/iOS/ViewExtensions.cs

```diff
@@ -32,10 +32,14 @@ public static void UpdateBackgroundColor(this UIView nativeView, IView view)
 		
 		public static void UpdateSemantics(this UIView nativeView, IView view)
 		{
-			nativeView.AccessibilityLabel = view.Semantics.Description;
-			nativeView.AccessibilityHint = view.Semantics.Hint;
+			var semantics = view.Semantics;
+			if (semantics == null)
+				return;
```

Comment:
This is weird, I don't think anything changed with this, but still seemed affected by the reordering of props.